test/spec/mapbox_logo.js

```diff
@@ -0,0 +1,83 @@
+describe('mapbox_logo', function() {
+    var server, element, doc, tileJSON = helpers.tileJSON;
+
+    beforeEach(function() {
+        server = sinon.fakeServer.create();
+        element = document.createElement('div');
+    });
+
+    afterEach(function() {
+        server.restore();
+    });
+
+    it('constructor', function() {
+        var map = L.mapbox.map(element, tileJSON);
+        var mapboxLogoControl = map._mapboxLogoControl;
+        expect(map._mapboxLogoControl instanceof L.Control);
+        expect(map.getTileJSON()).to.eql(helpers.tileJSON);
```

Comment:
Remove the assertions about `map.getTileJSON()` -- the behavior of `getTileJSON()` is not pertinent to MapboxLogoControl tests.